build_rpm.sh

```diff
@@ -65,6 +55,17 @@ do
   esac
 done
 
+case ${ARCH} in
+  osx*)
+    # Darwin is not RPM based, explicitly go for guessing the triplet
+    CONFIG_BUILD=guess
+    ;;
+  *)
+    # Assume Linux distro is RPM based and fetch triplet from RPM
+    CONFIG_BUILD=auto
+    ;;
+esac
+
```

Comment:
Why? This look wrong. Now this overrides `--build <type>` flag. E.g., I cannot use `--build guess` on Linux as it sets it back to `auto`.